platform/mingw64-linux/dependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+apt update -y
+apt install -y mingw-w64-x86-64-dev
+apt install -y autoconf libtool
+apt install -y bash-completion
+apt install -y build-essential
+apt install -y gcc-mingw-w64
+apt install -y g++-mingw-w64
+apt install -y git
+apt install -y mingw
+apt install -y mingw-w64-tools
+apt install -y vim
+apt install -y cmake
+apt install -y curl
+apt install -y zip
+apt install -y flex bison
+apt install -y wine
```

Comment:
Have you checked which of these are already installed on the github action runner? I suspect most are so this might be unnecessary. It might be useful for local runs though. I would avoid calling it in the CI if it's not needed.

Reply:
I updated the the ci to avoid installing the dependencies.